src/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs

```diff
@@ -214,47 +204,6 @@ private void CompileQilToMsil(XsltSettings settings)
         }
 
         //------------------------------------------------
-        // Compile stylesheet to a TypeBuilder
-        //------------------------------------------------
-
-        private static volatile ConstructorInfo s_generatedCodeCtor;
-
-        internal static CompilerErrorCollection CompileToType(XmlReader stylesheet, XsltSettings settings, XmlResolver stylesheetResolver, bool debug, TypeBuilder typeBuilder, string scriptAssemblyPath)
-        {
-            if (stylesheet == null)
-                throw new ArgumentNullException(nameof(stylesheet));
-
-            if (typeBuilder == null)
-                throw new ArgumentNullException(nameof(typeBuilder));
-
-            if (settings == null)
-                settings = XsltSettings.Default;
-
-            if (settings.EnableScript && scriptAssemblyPath == null)
-                throw new ArgumentNullException(nameof(scriptAssemblyPath));
-
-            if (scriptAssemblyPath != null)
-                scriptAssemblyPath = Path.GetFullPath(scriptAssemblyPath);
-
-            QilExpression qil;
-            CompilerErrorCollection errors = new Compiler(settings, debug, scriptAssemblyPath).Compile(stylesheet, stylesheetResolver, out qil).Errors;
-
-            if (!errors.HasErrors)
-            {
-                // Mark the type with GeneratedCodeAttribute to identify its origin
-                if (s_generatedCodeCtor == null)
-                    s_generatedCodeCtor = typeof(GeneratedCodeAttribute).GetConstructor(new Type[] { typeof(string), typeof(string) });
-
-                typeBuilder.SetCustomAttribute(new CustomAttributeBuilder(s_generatedCodeCtor,
-                    new object[] { typeof(XslCompiledTransform).FullName, Version }));
-
-                new XmlILGenerator().Generate(qil, typeBuilder);
-            }
-
-            return errors;
-        }
-
-        //------------------------------------------------
         // Load compiled stylesheet from a Type
         //------------------------------------------------
 
```

Comment:
The GeneratedCodeAttribute lives in System.Runtime and is part of NS2.0 so that is fine to keep.